shieldlib/package.json

```diff
@@ -41,12 +41,12 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/ZeLonewolf/openstreetmap-americana.git"
+    "url": "git+https://github.com/osm-americana/openstreetmap-americana.git"
```

Comment:
Remember to publish a new release of this package. Otherwise, I have no idea if NPM can cope with the repository getting renamed.